CONTRIBUTING.md

```diff
@@ -134,6 +134,21 @@ If you encounter the `gyp: No Xcode or CLT version` error on macOS at this step,
 
 ### Fix native modules to match current electron node version
 
+Before running the npm build scripts
+
+1. If you are using nvm to manage your Node installation on MacOS, you may have to add the following to a `~/.huskyrc` file:
+
+```
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+```
+
+2. You must specify a GitHub Personal Access Token. The error indicates "programmatically" is an option, but `gh auth login` did not solve the error. This can also be set as an environment variable (note your personal access tokens are sensitive!) [Read More About Personal Access Tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
```

Comment:
This is not mandatory. If you are hitting the issue about signing while building locally, then you can simply set `CSC_IDENTITY_AUTO_DISCOVERY` to `false` - which is already described in this same document.

Reply:
Thanks for the reference! I will update this note to indicate both options.

Reply:
Done